app/view_models/admin/edit_auction_view_model.rb

```diff
@@ -14,24 +14,44 @@ def new_record?
   end
 
   def hour_default(field)
-    dc_time(field).strftime("%l").strip
+    if field_value(field).present?
+      dc_time(field).strftime("%l").strip
+    else
+      "1"
```

Comment:
I almost wonder if it makes sense to just call the NewAuctionViewModel for these, but I suppose a little duplication is fine too